app/controllers/Application.java

```diff
@@ -118,9 +142,25 @@
   public static final String COMPARE_FLOW_ID1 = "flow-exec-id1";
   public static final String COMPARE_FLOW_ID2 = "flow-exec-id2";
   public static final String PAGE = "page";
+  public static final String SESSION_ID_KEY = "session_id";
```

Comment:
Access can be private , same is true at other places

Reply:
Some imports are being also used in the RestAPITest class so their access was made public. Also in commit 5ed16ab2f46e2d6c42fec7acfc170bf0ad40c179 these Constants are shifted to more suitable class.